dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/DataRequestLogger.kt

```diff
@@ -74,33 +72,4 @@ class DataRequestLogger {
         }
         logger.info(logMessage)
     }
-
-    /**
-     * Logs an appropriate message when a bulk data request notification mail shall be sent.
```

Comment:
Is this now part of the basic email logging?

Reply:
Yes. EmailSender manages this now.

Reply:
ok